test/e2e/clusterclass_changes_test.go

```diff
@@ -46,3 +46,29 @@ var _ = Describe("When testing ClusterClass changes [ClusterClass]", func() {
 		})
 	})
 })
+
+var _ = Describe("When testing ClusterClass changes [vcsim] [supervisor] [PR-Blocking] [ClusterClass]", func() {
+	const specName = "clusterclass-changes" // copied from CAPI
+	Setup(specName, func(testSpecificSettingsGetter func() testSettings) {
+		capie2e.ClusterClassChangesSpec(ctx, func() capie2e.ClusterClassChangesSpecInput {
+			return capie2e.ClusterClassChangesSpecInput{
+				E2EConfig:             e2eConfig,
+				ClusterctlConfigPath:  testSpecificSettingsGetter().ClusterctlConfigPath,
+				BootstrapClusterProxy: bootstrapClusterProxy,
+				ArtifactFolder:        artifactFolder,
+				SkipCleanup:           skipCleanup,
+				Flavor:                testSpecificSettingsGetter().FlavorForMode("topology"),
+				PostNamespaceCreated:  testSpecificSettingsGetter().PostNamespaceCreatedFunc,
+				ModifyControlPlaneFields: map[string]interface{}{
+					"spec.machineTemplate.nodeDrainTimeout": "10s",
+				},
+				ModifyMachineDeploymentBootstrapConfigTemplateFields: map[string]interface{}{
+					"spec.template.spec.verbosity": int64(4),
```

Comment:
q: This field now triggers a rotation which seems un-necessary, should we add this to the list of fields propagated in place?  (cc @sbueringer @chrischdi)
Might be there are other fields too...

Reply:
So this is kubeadms `-v` flag at the end? Makes sense to have this in-place 👍

Reply:
Also then: so we should propably pick a different field for this test?!

Reply:
Based on comments above, I would keep this field to intentianlly trigger a Machine rollout

Reply:
Should we really start patching KubeadmConfigs in-place for existing Machines in the Machine controller?

I get why the verbosity doesn't matter, but technically all fields of KubeadmConfig don't really matter, because kubeadm is only run once.

Reply:
taking my words back.
to change kubeadm verbosity we actually need a rollout